core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -676,4 +677,29 @@ default EventExecutor defaultSubscriberExecutor() {
         requireNonNull(function, "function");
         return new RecoverableStreamMessage<>(this, function, /* allowResuming */ true);
     }
+
+    /**
+     * Writes this {@link StreamMessage} to the given {@link Path} with {@link OpenOption}s.
```

Comment:
Could you copy the Javadoc about `OpenOption`?
```
     * If the {@link OpenOption} is not specified, defaults to {@link StandardOpenOption#CREATE},
     * {@link StandardOpenOption#TRUNCATE_EXISTING} and {@link StandardOpenOption#WRITE}.
```

Reply:
Sure! I'll put in.